### PR TITLE
feat: Privacy notice, Terms of Use, update Impressum, Cookies policy

### PR DESCRIPTION
**Pull request by DiogoSoaress**
- [x] Revert changes to `.github/workflows/deploy.yml`

Resolves https://github.com/safe-global/safe-homepage/issues/205

- [x] Privacy Notice
- [x] Terms of Use
- [x] Update Impressum
- [x] Cookie page

### How to test
You can test the implemented pages in https://safe-web-landing.staging.5afe.dev/

### Screenshots
Screenshot 2023-06-23 at 11 41 25

_expanding section_
Screenshot 2023-06-23 at 11 41 35

### CODE REVIEW

File: src/config/routes.ts [hunk 1]
```diff
@@ -3,11 +3,11 @@ export const AppRoutes = {
   wallet: '/wallet',
   terms: '/terms',
   privacy: '/privacy',
+  cookies: '/privacy#cookies',
   index: '/',
   imprint: '/imprint',
   ecosystem: '/ecosystem',
   core: '/core',
-  cookie: '/cookie',
```
Comment (katspaugh) [inline src/config/routes.ts:10]: Routes are generated automatically via a command in package json, don't edit it manually.
If you're removing this route, remove it from the pages folder.

File: src/components/common/CookieBanner/index.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const CookieBanner = (): ReactElement | null => {
       <div className={css.content}>
         <Typography align="center">
           We use cookies to provide you with the best experience and to help improve our website and application. Please
-          read our <Link href={AppRoutes.cookie}>Cookie Policy</Link> for more information. By clicking &quot;Accept
+          read our <Link href={AppRoutes.cookies}>Cookie Policy</Link> for more information. By clicking &quot;Accept
```
Comment (katspaugh) [inline src/components/common/CookieBanner/index.tsx:43]: Use `${AppRoutes.privacy}#cookied` instead.

File: src/components/common/CookieBanner/index.tsx [hunk 3]
```diff
@@ -40,9 +38,9 @@ export const CookieBanner = (): ReactElement | null => {
       <div className={css.content}>
         <Typography align="center">
           We use cookies to provide you with the best experience and to help improve our website and application. Please
-          read our <Link href={AppRoutes.cookie}>Cookie Policy</Link> for more information. By clicking &quot;Accept
-          all&quot;, you agree to the storing of cookies on your device to enhance site navigation, analyze site usage
-          and provide customer support.
+          read our <a href="https://app.safe.global/cookie">Cookie Policy</a> for more information. By clicking
```
Comment (katspaugh) [inline src/components/common/CookieBanner/index.tsx:41]: Pretty sure this is wrong.

**Comment by DiogoSoaress** [comment]
Revert changes made to `.github/workflows/deploy.yml` before merging into `main`

**Review by katspaugh — APPROVED** [review]

**Pull request merged**